src/patternfly/components/Drawer/drawer.scss

```diff
@@ -1,26 +1,36 @@
 $pf-c-drawer--breakpoint-map: build-breakpoint-map("base", "lg", "xl", "2xl");
-$pf-c-drawer__panel--width-list: (25, 33, 50, 66, 75);
+$pf-c-drawer-width--breakpoint-map: build-breakpoint-map("base", "lg", "xl", "2xl");
```

Comment:
that might be better as `$pf-c-drawer--breakpoint-map--width` (with "width" as a modifier/variation of "breakpoint-map")